solutions/usaco-815.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: usaco-815
+source: USACO Gold 2018 Febuary
+title: Taming the Herd
+author: Shijie Ren
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_gold_feb18.html)
+
+This solution is $\mathcal{O}(N^2)$ which is faster than the official editorial's.
+
+But not required since $N \leq 100$.
```

Comment:
```suggestion
This solution is $\mathcal{O}(N^2)$ which is faster than the official editorial's but not required since $N \leq 100$.
```

Reply:
feel like a comma should be put between N^2 and which